README.md

```diff
@@ -102,6 +102,12 @@ $ ./basic -o foo.txt -dvvvs 1337 -l alice -l bob --nb-cars 4 bar.txt baz.txt
 Opt { debug: true, verbose: 3, speed: 1337, output: "foo.txt", nb_cars: Some(4), level: ["alice", "bob"], files: ["bar.txt", "baz.txt"] }
 ```
 
+## StructOpt rustc version policy
+
+- Minimum rustc version modification must be specified in the [changelog](https://github.com/TeXitoi/structopt/blob/master/CHANGELOG.md) and in the [travis configuration](https://github.com/TeXitoi/structopt/blob/master/.travis.yml).
+- Contributors can increment minimum rustc version without any justification if the new version is required by the latest version of one of StructOpt's depedencies (`cargo update` will not ffail on StructOpt).
+- Contributors can increment minimum rustc version if the library user experience is improved.
+
```

Comment:
So it's cool to bump the min required rustc version between structopt 1.2.3 to 1.2.4? I'd probably try and do that in minor versions only. This is also the policy in some other crates

Reply:
Right now, we are in 0.x version, thus I don't want to do a breaking change version to upgrade rustc.

But, after 1.0.0 I think that following the minor rule would bethe way to go.